dev-packages/node-integration-tests/suites/tracing/genericPool/scenario.js

```diff
@@ -0,0 +1,71 @@
+const { loggingTransport } = require('@sentry-internal/node-integration-tests');
+const Sentry = require('@sentry/node');
+
+Sentry.init({
+  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  release: '1.0',
+  tracesSampleRate: 1.0,
+  transport: loggingTransport,
+});
+
+// Stop the process from exiting before the transaction is sent
+setInterval(() => {}, 1000);
+
+const mysql = require('mysql');
```

Comment:
The resource used for `generic-pool` can be anything. But I picked `mysql` for its simple setup.